.gitattributes

```diff
@@ -20,4 +20,7 @@
 *.ico binary
 *.snk binary
 *.pdb binary
-*.svg binary
\ No newline at end of file
+*.svg binary
+
+# Don't check for whitespace errors in the doc pages
+*.md -whitespace
```

Comment:
Wouldn't it be better for force line normalisation? What does this achieve, and does github diff take this into account?